js/therock.js

```diff
@@ -351,6 +351,203 @@ module.exports = class therock extends Exchange {
         };
     }
 
+    async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {
+        return this._fetchTransactions ('withdraw', code, since, limit, params);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        return this._fetchTransactions ('atm_payment', code, since, limit, params);
+    }
+
+    async _fetchTransactions (type, code = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
This is definitely a `fetchLedger`, not `fetchTransactions`, I will have to rework this.

Reply:
I'd say that it could be fetchLedger, but for now it's fetchTransactions.

Reply:
@npomfret plz check this out: https://github.com/ccxt/ccxt/pull/4179#discussion_r234361900

Reply:
Also this is important: https://github.com/ccxt/ccxt/pull/4161#issuecomment-438710176

Reply:
@npomfret i will add the edits to this PR to rewire the unified methods. If you can then take a look at the edits I made and reflect them in further PRs – that would really speed up the process  ) but if not, your help and effort is still very much appreciated ) thx )